string age
string gender
string up_color